### PR TITLE
feat: add an entry to the feedstore, when adding to the state store

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: packages/core/src/state-management/repository.ts [hunk 1]
```diff
@@ -336,12 +336,17 @@ export class Repository {
...
     // TODO (NET-1687): unify shouldIndex check into indexStreamIfNeeded
     const shouldIndex =
       state$.state.metadata.model && this.index.shouldIndexStream(state$.state.metadata.model)
     if (isPinned || shouldIndex) {
-      await this.pinStore.add(state$)
+      await this._pin_UNSAFE(state$)
+      await this.feedAggregationStore.put(state$.id)
+      // const pinStoreAdd = this._pin_UNSAFE(state$)
+      // const feedStorePut = this.feedAggregationStore.put(state$.id)
+      // await Promise.all([pinStoreAdd, feedStorePut])
```
Comment (JulissaDantes) [inline packages/core/src/state-management/repository.ts:349]:
```suggestion
```

**Review by JulissaDantes — COMMENTED** [review]
Only a few comments, but overall I think everything looks pretty straightforward.

**Comment by ukstv** [comment]
thank you @JulissaDantes !!

**Review by JulissaDantes — APPROVED** [review]
LGTM!

**Pull request merged**